test/cmdlineTests/ast_compact_json_no_pretty_json/input.sol

```diff
@@ -0,0 +1,4 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity >=0.0;
+
+import "./c.sol";
```

Comment:
I'd remove `c.sol` to simplify the test. It was relevant in the test with base path but for pretty printing we don't need it.